dotnet/VERSIONS.md

```diff
@@ -4,7 +4,7 @@
 
 Our NuGet packages are versioned using [Semver 2.0.0][2].
 
-This is the scheme: `OsMajor.OsMinor.InternalRelease[-prereleaseX]+sha.1b2c3d4`.
+This is the scheme: `OsMajor.OsMinor.InternalRelease[-prereleaseX].CommitDistance`.
```

Comment:
This is not a valid semver version - semver only allows three numbers, not four: 

```suggestion
This is the scheme: `OsMajor.OsMinor.InternalRelease[-prereleaseX]`.
```